lib/standards/html-elms.js

```diff
@@ -329,6 +329,31 @@ const htmlElms = {
   },
   img: {
     variant: {
+      noAlt: {
+        matches: {
+          hasAccessibleName: true,
+          attributes: {
+            alt: null
+          }
+        },
```

Comment:
I think this should let us get rid of the second variant.

```suggestion
        matches: [{
          // Because <img role="none" alt="foo" /> has no accessible name:
          attributes: {
            alt: '/.+/'
          }
        }, {
          hasAccessibleName: true,
        }],
```